control_plane_readme.md

```diff
@@ -98,6 +125,41 @@ Directly access your postgres database:
 psql postgres://postgres:postgres@localhost:5432/postgres
 ```
 
+### EXPERIMENTAL Start private PG (instead of Supabase for local development):
+
+Install and start pg:
```

Comment:
My 2 cents is that it's easier to manage local postgres databases using docker. I don't think it really matters how you install postgres, though, so I don't think it would hurt anything to have this section :man_shrugging:

Reply:
That was to run it without docker. Anyways, dropped pg install steps in the patch below.